neal/sampler.py

```diff
@@ -97,75 +99,95 @@ def __init__(self):
+               initial_states=None, initial_states_generator="random",
+               **kwargs):
+        """Sample from a binary quadratic model using an implemented sample 
+        method.
 
         Args:
             bqm (:class:`dimod.BinaryQuadraticModel`):
                 The binary quadratic model to be sampled.
 
-            beta_range (tuple, optional):
+            beta_range (tuple, optional, default = None):
                 A 2-tuple defining the beginning and end of the beta schedule,
                 where beta is the inverse temperature. The schedule is
                 interpolated within this range according to the value specified
-                by `beta_schedule_type`. Default range is set based on the total
-                bias associated with each node.
+                by ``beta_schedule_type``. Default range is set based on the 
+                total bias associated with each node.
 
             num_reads (int, optional, default=len(initial_states) or 1):
-                Number of reads. Each read is generated by one run of the simulated
-                annealing algorithm. If `num_reads` is not explicitly given, it is
-                selected to match the number of initial states given. If initial states
-                are not provided, only one read is performed.
-
-            num_sweeps (int, optional, default=1000):
-                Number of sweeps or steps.
-
-            beta_schedule_type (string, optional, default='geometric'):
-                Beta schedule type, or how the beta values are interpolated between
-                the given 'beta_range'. Supported values are:
-
-                * linear
-                * geometric
-
-            seed (int, optional):
-                Seed to use for the PRNG. Specifying a particular seed with a constant
-                set of parameters produces identical results. If not provided, a random seed
-                is chosen.
+                Number of reads. Each read is generated by one run of the 
+                simulated annealing algorithm. If `num_reads` is not explicitly
+                given, it is selected to match the number of initial states 
+                given. If initial states are not provided, only one read is 
+                performed.
+
+            num_sweeps (int, optional, default=``len(beta_schedule)*num_sweeps_per_beta`` or 1000):
+                Number of sweeps used in annealing. If no value is provided
+                and ``beta_schedule`` is None the value is defaulted to 1000.
+
+            num_sweeps_per_beta (int, optional, default=1)
+                Number of sweeps to perform at each beta. One sweep consists of
+                a sequential Metropolis update of all spins.
+
+            beta_schedule_type (string, optional, default="geometric"):
+                Beta schedule type, or how the beta values are interpolated 
+                between the given `beta_range`. Supported values are:
```

Comment:
Needs an empty line between the paragraph and the list (otherwise the rendered text will be "Supported values are: \* "linear" \* "geometric" \* "custom"